### PR TITLE
sean_cypress

### PR DESCRIPTION
**Pull request by Sean1783**
Fixed 'remove' function. Added real-time tallying. Experimenting with displaying records on search.

### CODE REVIEW

**Review by andrewjunggg — CHANGES_REQUESTED** [review]
Hello Sean,
Don't forget to update the branch with the latest main!

**Comment by Sean1783** [comment]
> Hello Sean, Don't forget to update the branch with the latest main!
Thanks dude - should be all set now.

File: backend/index.js
Comment (andrewjujun) [inline backend/index.js:1]: Don't forget to change back to `*` so it works on the server!

File: src/app/search/page.js
Comment (andrewjujun) [inline src/app/search/page.js:1]: Same here!

File: src/components/IndividualTotals/IndividualTotals.js [hunk 1]
```diff
@@ -1,82 +1,39 @@
...
-  //     tips: props.tips
-  //   };
-
-  
 
     try {
       const response = await fetch(
-        // 'http://localhost:3333/sean/createBill'
-        'https://gpt-billsplitter.com:3333/sean/createBill',
+        'http://localhost:3333/sean/createBill'
```
Comment (andrewjujun) [inline src/components/IndividualTotals/IndividualTotals.js]: Here too!

**Review by andrewjunggg — CHANGES_REQUESTED** [review]

**Review by andrewjunggg — APPROVED** [review]
Looks good now.

**Pull request merged**